src/execCli.js

```diff
@@ -72,7 +73,7 @@ export async function execCli(argv) {
   try {
     cli.parse(argv, { run: false })
     await cli.runMatchedCommand()
-    logger.success(`Done in ${timer.formatElapsedTime()}`)
+    logger.log(`${padString('Time:')} ${timer.formatElapsedTime()}\n`)
```

Comment:
let's remove this and the `padString` function, then move the timer and the `Time:` label into the `run` command. It's not so useful for the other tasks.

Reply:
Done 👍